release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -550,6 +550,18 @@ For information about using the {secrets-store-operator} to mount secrets from a
 
 For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file-nfs_persistent-storage-csi-azure-file[NFS support].
 
+[id="ocp-4-14-oci"]
+=== Oracle&#174; Cloud Infrastructure
+
+You can now install an {product-title} cluster on {oci} by using either the Assisted installer or the Agent-based installer. For {product-title} {product-version}, {product-title} on {oci} is available as a Developer Preview feature.
+
+To install an {product-title} cluster on {oci}, choose one of the following installation options:
+
+* link:https://access.redhat.com/articles/7039183[Using the Assisted Installer to install a cluster on {oci-first}]
```

Comment:
These  2 links do not resolve. Are they supposed to resolve now or after GA (release notes)?

Reply:
HI, @tmalove . Correct. The article are not published just yet. They must wait until GA.

Reply:
I can see the drafts, they are the correct links :+1: